### PR TITLE
Introduce renamed functions/variables and clustering

### PR DESCRIPTION
**Pull request by StefanBrand**
This PR includes the other PR I have made from the branch ´renaming´. I'm not sure how it will behave. :smile: 

PS.: Here is a demo of the clustering: https://stefanbrand.github.io/fprsmap/ I have also tested it with two markers on different layers that were on the exact same location. Worked like a charm! :smiley:

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -38,22 +38,23 @@
 
   /* Variables (state) */
   var map;
+  var cluster = L.markerClusterGroup();
```
Comment (Roboe) [inline js/main.js]: We don't need `cluster` to be broadly available, thus I suggest to move the initialization to `initMap()`, right after `baseLayer` initialization for not polluting the global scope.
If we need it later, we can move it back.

File: index.html [hunk 2]
```diff
@@ -35,10 +35,14 @@
 
 	<!-- Styles -->
 	<link rel="stylesheet" href="https://bowercdn.net/c/leaflet-0.7.7/dist/leaflet.css" integrity="sha384-E9vFwWyHrMFVHKrPgvBjKuVepLDtu+A/vJWwKWrQJ4nZEGbuwutpoM6VNO/nc9dm" crossorigin="anonymous">
-	<link rel="stylesheet" href="css/style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet.markercluster/0.5.0/MarkerCluster.Default.css">
```
Comment (Roboe) [inline index.html]: Tab here, not spaces. We are using tabs on HTML files and spaces on JS

**Comment by Roboe** [comment]
I created the branch conflict, I'll solve it, hahaha (just a thorought `git rebase -i`)

File: index.html (hunk 2 above)
Comment (StefanBrand) [inline index.html]: Looks good in my Atom. I think it was indented automatically...

File: js/main.js (hunk 1 above)
Comment (StefanBrand) [inline js/main.js]: Don't we need it, when we create `overlaysData`?

**Comment by Roboe** [comment]
Do I have push permissions? :/
(or "Is the `Allow edits from maintainers` check enabled?")

**Comment by StefanBrand** [comment]
Don't mess up! :wink:

**Comment by Roboe** [comment]
Hahaha, I'll never force-push to a non-owned PR again! Anyway, I fixed it [manually](https://github.com/WeAreFairphone/fprsmap/commit/eb8f35485716c779b5fca097afc3f9f7a8ddab1d) (<3 the command line)

**Comment by StefanBrand** [comment]
Maybe it's not working perfectly yet, if they have just recently implemented the maintainer-thing.

**Pull request closed without merging**